gulpfile.js

```diff
@@ -16,7 +16,8 @@ var appDir = srcDir + '/app';
 
 var paths = {
   sass: [appDir + '/style/**/*.scss'],
-  scripts: [appDir + '/**/*.module.js', appDir + '/**/*.js', '!' + appDir + '/**/*.spec.js', appDir + '/app.js']
+  scripts: [appDir + '/**/*.module.js', appDir + '/**/*.js', '!' + appDir + '/**/*.spec.js', appDir + '/app.js'],
+  html: [appDir + '/**/*.html']
```

Comment:
Guarda la lista dei commit:
https://github.com/the-software-factory/angular-darts-tournament/pull/118/commits/aba4eacbbb5686ca5473f439b1128a6921493918 deve andare via :)